<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(10, 11, 12);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>961</width>
      <height>541</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>200</y>
        <width>241</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(3, 252, 73);
color:white;
font-family:Lalezar;
font-size:32px;
border:none;
border-radius:20px;</string>
      </property>
      <property name="text">
       <string>مدرس هستم</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>80</y>
        <width>461</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
font-family:Lalezar;
font-size:44px;</string>
      </property>
      <property name="text">
       <string>تست بانک مرجع سوالات آزمونی</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>200</y>
        <width>241</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(3, 252, 73);
color:white;
font-family:Lalezar;
font-size:32px;
border:none;
border-radius:20px;</string>
      </property>
      <property name="text">
       <string>دانشجو هستم</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>190</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(3, 252, 73);
color:white;
font-family:Lalezar;
font-size:26px;
border:none;
border-radius:30px;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>نام کاربری</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>270</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(3, 252, 73);
color:white;
font-family:Lalezar;
font-size:26px;
border:none;
border-radius:30px;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>رمز عبور</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>350</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(3, 252, 73);
color:white;
font-family:Lalezar;
font-size:26px;
border:none;
border-radius:30px;
padding-left:-10px;</string>
      </property>
      <property name="text">
       <string>بعدی</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>100</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
font-family:Lalezar;
font-size:48px;</string>
      </property>
      <property name="text">
       <string>ثبت نام در سامانه</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>440</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:none;
color:white;
font-family:Lalezar;
font-size:20px;
border:none;
border-radius:30px;</string>
      </property>
      <property name="text">
       <string>ورود  نام در تست بانک</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>480</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(3, 252, 73);
font-family:Lalezar;
font-size:20px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_17">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>350</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(3, 252, 73);
color:white;
font-family:Comic SANS MS;
font-size:44px;
border:none;
padding-top:-7px;
border-radius:30px;</string>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>350</y>
        <width>321</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(3, 252, 73);
color:white;
font-family:Lalezar;
font-size:26px;
border:none;
border-radius:30px;
padding-left:-10px;</string>
      </property>
      <property name="text">
       <string>بعدی</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>100</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
font-family:Lalezar;
font-size:48px;</string>
      </property>
      <property name="text">
       <string>ورود به سامانه</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>270</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(3, 252, 73);
color:white;
font-family:Lalezar;
font-size:26px;
border:none;
border-radius:30px;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>رمز عبور</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>190</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(3, 252, 73);
color:white;
font-family:Lalezar;
font-size:26px;
border:none;
border-radius:30px;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>نام کاربری</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>440</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:none;
color:white;
font-family:Lalezar;
font-size:20px;
border:none;
border-radius:30px;</string>
      </property>
      <property name="text">
       <string>ثبت نام در تست بانک</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_16">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>350</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(3, 252, 73);
color:white;
font-family:Comic SANS MS;
font-size:44px;
border:none;
padding-top:-7px;
border-radius:30px;</string>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_10">
     <widget class="QListWidget" name="listWidget_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>721</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
font-family:Lalezar;
font-size:30px;
padding-top:20px;
border-radius:20px;
border:2px solid rgb(3, 252, 73);</string>
      </property>
      <property name="itemAlignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>430</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(3, 252, 73);
color:white;
font-family:Comic SANS MS;
font-size:44px;
border:none;
padding-top:-7px;
border-radius:30px;</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>721</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
font-family:Lalezar;
font-size:30px;
padding-top:20px;
border-radius:20px;
border:2px solid rgb(3, 252, 73);</string>
      </property>
      <property name="itemAlignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>270</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(3, 252, 73);
color:white;
font-family:Lalezar;
font-size:26px;
border:none;
border-radius:30px;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>پاسخ سوال</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>190</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(3, 252, 73);
color:white;
font-family:Lalezar;
font-size:26px;
border:none;
border-radius:30px;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>صورت سوال</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>350</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(3, 252, 73);
color:white;
font-family:Lalezar;
font-size:26px;
border:none;
border-radius:30px;
padding-left:-5px;</string>
      </property>
      <property name="text">
       <string>ثبت</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>100</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
font-family:Lalezar;
font-size:48px;</string>
      </property>
      <property name="text">
       <string>سوال طرح کن</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>440</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(3, 252, 73);
font-family:Lalezar;
font-size:20px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>350</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(3, 252, 73);
color:white;
font-family:Comic SANS MS;
font-size:44px;
border:none;
padding-top:-7px;
border-radius:30px;</string>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>848</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
